test/e2e/camoufox-cloudflare/actor/Dockerfile

```
@@ -0,0 +1,23 @@
+FROM apify/actor-node-playwright-chrome:22-beta AS builder
+
+COPY /packages ./packages
+COPY /package*.json ./
+RUN npm --quiet set progress=false \
+    && npm install --only=prod --no-optional --no-audit \
+    && npm update
+
+FROM apify/actor-node-playwright-chrome:22-beta
+
+RUN rm -r node_modules
+COPY --from=builder /node_modules ./node_modules
+COPY --from=builder /packages ./packages
```

```suggestion
COPY --from=builder /node_modules ./node_modules
ENV PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=0
RUN npm run postinstall
COPY --from=builder /packages ./packages
```

After the latest changes to `camoufox-js`, you need first to enable the browser download again. And since we don't run the `npm install` script in the second stage, you also need to invoke the `postinstall` script directly.

There might be a better order to these steps, though (in terms of Docker cache usage). 

Alternative idea: Camoufox installs the browser to `[homedir] / .cache / camoufox` ([see here](https://github.com/apify/camoufox-js/blob/20a4601b34058d89d99343ac4f11a90df9bd283d/src/pkgman.ts#L40)), maybe we can download the browser in the builder stage already and just copy it over from there?